### PR TITLE
feat(dc-collective-alert): update styles to match design

### PR DESCRIPTION
**Pull request by arathjz**
**What:**
Update debt collective alert styles

**Why:**
To address: https://app.asana.com/0/1159164196409129/1186341795801831/f

**How:**
- Change the position of button and label
- Update colors to match the design
- Add join icon to theme assets

### Media
https://www.loom.com/share/2880b22b485044d0aca1a03a112a286e

### CODE REVIEW

File: about.json [hunk 1]
```diff
@@ -3,7 +3,9 @@
   "about_url": null,
   "license_url": null,
   "assets": {
-    "hero_image": "assets/handshake.png"
+    "hero_image": "assets/handshake.png",
+    "dc_allert_icon_blue": "assets/icons/dc-allert-icon-blue.svg",
```
Comment (duranmla) [inline about.json:7]: Icons we use often are listed at https://material.io/resources/icons/?style=baseline try to use one of them or to find something similar. How to use it?

```
<span class="material-icons">
    comment
</span>
````

The above will render the comment icon in a font based way _(so you can change the colour or size easily)_

Comment (arathjz) [reply about.json:7]: Using an icon might not be possible due the button is a discourse component

```hbs
{{d-button action=(action "join")
    class="btn-alert"
    label="discourse_debtcollective_collectives.join"
    disabled=disabled}}
```

Do you know if I can pass something like `children` to this discourse component?

Comment (arathjz) [reply about.json:7]: Fixed 💯 @duranmla

File: javascripts/discourse/templates/components/dc-collective-alert.hbs [hunk 2]
```diff
@@ -1,13 +1,15 @@
 {{#unless hidden}}
 <div class="row">
   <div class="alert alert-info alert-collective-alert">
+    <button class="dc-alert-btn" {{action "join"}}>
```
Comment (duranmla) [inline javascripts/discourse/templates/components/dc-collective-alert.hbs:4]: 🔥

**Comment by orlando** [comment]
@arathjz we will need to clean up locales in the sso plugin too, no rush but just to keep that in mind

**Pull request merged**